lib/api/objectDelete.js

```diff
@@ -51,6 +51,13 @@ function objectDeleteInternal(authInfo, request, log, isExpiration, cb) {
     const reqVersionId = decodedVidResult;
     const hasGovernanceBypass = hasGovernanceBypassHeader(request.headers);
 
+    // Reassign a specific apiMethod, as monitoring can rely on the
```

Comment:
as far as I can tell (looking at the code), `apiMethod` is not used for monitoring: but mostly to
* override the default `requestType` passed to `standardMetadataValidateBucketAndObj()` in every api....
* in prepareRequestContexts().... but not for backbeat routes, which do the evaluation by assuming an api method of 'objectReplicate' 😱
* in the "standard" routes, to handle special handling of specific APIs (which may be needed for backbeat routes as welll...fortunately we don't seem to use these affected API -mostly GetObject/PutObject- through backbeat routes)